index.js

```diff
@@ -32,15 +32,51 @@ PLAYERS.forEach(renderPlayer)
 
 
 /***** Deliverable 1 *****/
+let pageHeader = document.querySelector("h1#header")
+
+pageHeader.addEventListener('click', (event) => {
+element = event.target
+toggleColor(element)
+})
+
+
 function toggleColor(element) {
   if (element.style.color === "red") {
+
     element.style.color = "black"
   } else {
     element.style.color = "red"
+
   }
 }
 
 
 /***** Deliverable 2 *****/
 
+const newPlayerForm = document.querySelector('form#new-player-form')
+newPlayerForm.addEventListener('submit', (event) =>{
+  event.preventDefault()
+  let playerObj = {
+    number: event.target.number.value,
+    name: event.target.name.value,
+    nickname: event.target.nickname.value,
+    photo: event.target.photo.value,
+    likes: 0
+  }
+  renderPlayer(playerObj)
+  event.target.reset()
+}
+)
+
 /***** Deliverable 3 *****/
+
+playerContainer.addEventListener('click',(event)=> { 
+  if (event.target.matches("button.like-button")){
+    let button = event.target
+    let player = button.closest("div.player")
+    let likes = player.querySelector(".likes")
+    let playerLikes = parseInt(likes.innerText) + 1
+
+    likes.innerHTML = playerLikes
```

Comment:
Since you're just adding text inside the `likes` element, and not adding HTML, you can just use `.innerText` or `.textContent` instead of `.innerHTML`. You can also do some string interpolation to keep the text consistent:

```js
likes.textContent = `${playerLikes} likes`
```